x-pack/plugins/fleet/common/services/agent_status.ts

```diff
@@ -41,36 +41,42 @@ export function getAgentStatus(agent: Agent, now: number = Date.now()): AgentSta
   return 'online';
 }
 
-export function buildKueryForEnrollingAgents() {
-  return 'not (last_checkin:*)';
+export function buildKueryForEnrollingAgents(path: string = '') {
```

Comment:
adding an optional path to the fields since we copy .fleet-agents mapping but are additionally nesting it